scripts/provision.sh

```diff
@@ -32,6 +32,10 @@ configure() {
   export PGPASSWORD
   PGPASSWORD=$(cat /hab/svc/builder-datastore/config/pwfile)
 
+  export ANALYTICS_ENABLED=${ANALYTICS_ENABLED:="false"}
+  export ANALYTICS_COMPANY_ID=${ANALYTICS_COMPANY_ID:="builder-on-prem"}
+  export ANALYTICS_WRITE_KEY=${ANALYTICS_WRITE_KEY:="NAwVPW04CeESMW3vtyqjJZmVMNBSQ1K1"}
```

Comment:
Now that I look at this, I'll bet by providing a write key, the API will send its events. I'm going to add a check to leave this value empty unless `ANALYTICS_ENABLED` is `true` just to prevent any inadvertent data-gathering.